dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14-alpine
+
+ARG BRANCH_NAME
```

Comment:
Can we define a a default arg value? for example `master` ? also document that if no branch is passed during building time then `master` will be used